lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -145,7 +144,10 @@
                                 message = response.user_message;
                             }
                             view.showMessage(message, message);
-                        });
+                        })
+                    ViewUtils.disableElementWhileRunning($(".action-primary"), function() {
+                        return deferred.promise();
```

Comment:
Don't you have the deferred.promise and deferred.resolve reversed? I would expect the method to return deferred.promise(), and disableElementWhileRunning to do the resolve inside of its function.

Is disableElementWhileRunning actually used anywhere in the codebase currently?